stable/prometheus/templates/alertmanager-pvc.yaml

```diff
@@ -3,6 +3,11 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   annotations:
+  {{- if .Values.alertmanager.persistentVolume.storageClass -}}
+    volume.beta.kubernetes.io/storage-class: {{ .Values.alertmanager.persistentVolume.storageClass }}
+  {{ else }}
+    volume.alpha.kubernetes.io/storage-class: ''
```

Comment:
According to http://kubernetes.io/docs/user-guide/persistent-volumes/#writing-portable-configuration we should set it to default (although I don't think it actually makes a difference)